src/test/java/org/gridsuite/shortcircuit/server/ShortCircuitAnalysisControllerTest.java

```diff
@@ -218,7 +218,7 @@ private static void assertFaultResultsEquals(List<FaultResult> faultResults, Lis
             assertEquals(orderedFeederResultsDto.size(), orderedFeederResults.size());
             for (int j = 0; j < orderedFeederResultsDto.size(); j++) {
                 assertEquals(orderedFeederResultsDto.get(j).getConnectableId(), orderedFeederResults.get(j).getConnectableId());
-                if (orderedFeederResults.get(i) instanceof MagnitudeFaultResult) {
+                if (faultResults.get(i) instanceof MagnitudeFaultResult) {
```

Comment:
This is a TU fix of a complete another scope. I would choose to push it in another PR.

Reply:
Ok I remove